### ISSUE TITLE
Unable to access tellstick.local

### ISSUE THREAD

**Issue by stefffe** [quazzie/tellstick-plugin-mqtt-hass#42]
Hey, after Telldus changed everything this autumn, I seem to be unable to login locally.
I have contacted the devs with the public key, and got it added to my Telldus account.
But when trying to access tellstick.local, I get to the Telldus login page, but when entering my credentials I face the attached image.
![IssueLoggingInLocal](https://github.com/quazzie/tellstick-plugin-mqtt-hass/assets/81174021/315cdc37-8c14-429f-8c64-6ffd6343424e)
Any ideas on how I can upload the plugin given this situation? Or do you need a premium account to do that?

**Comment by jptuomi** [comment]
I just used another browser (Safari in this case) and then it worked, try it out if possible.

**Comment by stefffe** [comment]
Wow, yeah, I had already tested with both Firefox and Chrome both in regular and private mode and it did not work, but when testing with Microsoft Edge everything worked just fine! Thanks! :o)
Plugin now successfully installed!

**Comment by hauard** [comment]
Any troubles with the plugin / znet disconnecting the broker, either immediately, or after some time?
I've had connection drops all the time, but now it justs won't stay connected. It connects, but after a few seconds, it disconnects.

**Comment by MaxwellErnst** [comment]
> Any troubles with the plugin / znet disconnecting the broker, either immediately, or after some time? I've had connection drops all the time, but now it justs won't stay connected. It connects, but after a few seconds, it disconnects.

Have you tried the lua script: https://github.com/quazzie/tellstick-plugin-mqtt-hass/issues/30#issuecomment-1520743711

**Comment by hauard** [comment]
> > Any troubles with the plugin / znet disconnecting the broker, either immediately, or after some time? I've had connection drops all the time, but now it justs won't stay connected. It connects, but after a few seconds, it disconnects.
> 
> Have you tried the lua script: [#30 (comment)](https://github.com/quazzie/tellstick-plugin-mqtt-hass/issues/30#issuecomment-1520743711)

Yes. The MQTT-connection is not alive long enough to publish or recieve any messages, so it will always time out and try to restart the connection.

I ended up using this Node-RED palette:
[Telldus NodeRED Contrib](https://github.com/perbrage/node-red-contrib-telldus-zwave-v2-local
)

Works almost like a charm! The only bug is I'm not getting on signals when device is already on, and vice versa.
